coralproject/talk
# [CORL-882] option to reject all a user's comments when banning

## What does this PR do?
- Adds a checkbox to the "ban user" modal to enable the option
- if checked, a job is queued to reject all the user's existing comments  

## How do I test this PR?
- create a user with one or more comments
- ban this user, selecting "reject all this users comments"
- check "rejected" queue to ensure comments are rejected
